ee/agent/knapsack/knapsack.go

```diff
@@ -87,6 +87,10 @@ func (k *knapsack) AgentFlagsStore() types.KVStore {
 	return k.getKVStore(storage.AgentFlagsStore)
 }
 
+func (k *knapsack) AtcConfigStore() types.KVStore {
```

Comment:
This will end up in a new bucket, right? Do we want that vs I-don't-know

Reply:
Yes, new bucket! I think the other option would be making it an agent flag and it'd live alongside the other agent flags, which I'm not opposed to either. A new bucket and subsystem felt like it gave me a little bit more room to iterate.

Reply:
<thinks> I'm game to try....

In a world where we have 4 katc tables, what will that look like? I imagine they'll be a DSL where we have a table name, and then some parameters defining it.

So if that was in the agent flags, we'd have `katc => { table1 => {...}, table2 => {...} }` and if it was in a bucket we'll see entries for each table? I guess it's kinda all the same.

Reply:
Yeah, the options are exactly as you describe. They both feel ok to me -- let's try this and it shouldn't be painful to switch if we change our minds later 🙂